### ISSUE TITLE
General .90 on Hard FeedBack

### ISSUE THREAD

**Issue by Pharylon** [RemoteTechnologiesGroup/RemoteTech#273]
This is really more a general feedback issue than a specific suggestion... but I feel like RemoteTech is "broken" when playing with Tier 1 Buildings. I started a new game set to Hard with RemoteTech, and it was... well... HARD. And not fun-hard. It was frustrating-hard (although, I admit, that's very subjective!). Issues I see are:

* Dishes are pretty much useless on Tier 1. They just take too much power unless you have the more advanced solar panels. I think scaling back power usage for Kerbin-SOI ranges might be necessary.

* No money. When money is tight, every mission needs to fulfil a contract, and there's no much room for error. Putting up a decent communications network is expensive, though. Maybe implementing some RemoteTech contracts? That'd be pretty cool. It could start with a "Sputnik" that just has to ping KSC for a few orbits and move up to a full interplanetary network. That would also really make it feel more integrated into the game. But, again, mainly, the issue is just how it feels like a time waster in the early game. Making building the communications network part of a contract would make it feel more fun.

* Kerbostationary orbit is useless. Some existing contracts already have you putting satellites in Kerbostationary orbit, which in real life is where communication satellites go. But the antenna lengths are set up such that Kerbostationary Orbit isn't very good. The default omni doesn't reach that high unless you use Root distances. That kind of sucks. I don't think Root should be the default distance measurement (it's complicated!) and I realize increasing the starting Omni gives it a pretty big range (although I think as long as it's less than Munar distances, it's functionally the same), but putting communication satellites in Kerbostationary orbit just "feels" right, and lets you piggy-back comsats on existing contracts, which would be fun!

**Comment by erendrake** [comment]
I agree and i think a lot of mods are tyring to figure out how to make hard mode work for them. thanks for the feedback :)
